cypress/integration/regrister.spec.js

```diff
@@ -1,148 +1,95 @@
-import signUpPage from "../fixtures/signUpPage.json";
-import loginPage from "../fixtures/loginPage.json";
 import data from "../fixtures/data.json";
-import sidebar from "../fixtures/sidebar.json";
-import navigation from "../fixtures/navigation.json";
+import faker from "faker";
+import authModule from "../models/authModule";
+import validationMessages from "../fixtures/validationMessages.json";
 
 describe("Register", () => {
+  let user = {
```

Comment:
This should probably be in the test setup (external module) as well, e.g. `userModule` etc